examples/aws-atlas-privatelink/variables.tf

```diff
@@ -0,0 +1,32 @@
+variable "public_key" {
+  description = "The public API key for MongoDB Atlas"
```

Comment:
Normally when I include any secrets I remind folks it's better to use a secret manager, etc and to be sure to git ignore any variables files in as many places as possible.  Again not a blocker but a nice-to-do for, especially new, users